ante/tests/setup.go

```diff
@@ -57,15 +56,14 @@ func createTestApp(t *testing.T, isCheckTx bool) (*simapp.SimApp, sdk.Context) {
 	return app, ctx
 }
 
-func TestAnteTestSuite(t *testing.T) {
-	suite.Run(t, new(AnteTestSuite))
-}
+// func TestAnteTestSuite(t *testing.T) {
+// 	suite.Run(t, new(AnteTestSuite))
+// }
 
 // SetupTest setups a new test, with new app, context, and anteHandler.
 func (s *AnteTestSuite) SetupTest(isCheckTx bool) {
-	s.app, s.ctx = createTestApp(s.T(), isCheckTx)
+	s.app, s.ctx = createTestApp(isCheckTx)
```

Comment:
Is this successful on the simapp bootstrap?

Reply:
Yes